### PR TITLE
Task3: cycles and conditions

### PR DESCRIPTION
**Pull request by IrynaKazei**
Please review my Task3

### CODE REVIEW

File: tasks/condition/condition.js [hunk 1]
```diff
@@ -34,5 +50,24 @@
 {
     const x = 3;
     const y = 7;
+    let result;
     const operator = "add";
+
+    switch (operator) {
+        case operator === "add":
```
Comment (dshaplyko) [inline tasks/condition/condition.js:57]: you can simply write:
case 'add':
...
and so on

File: tasks/cycles/cycles.js [hunk 2]
```diff
@@ -11,37 +11,85 @@
...
+
 /**
  * return concatenated string from an array of substring
  */
 {
-  const substr = ["I", " love", " JS"];
+    const substr = ["I", "love", "JS"];
+    let phrase= '';
+    for (let i = 0; i < substr.length; i++) {
+        phrase = phrase + substr[i] + " ";
```
Comment (dshaplyko) [inline tasks/cycles/cycles.js:72]: you can use the short form:
phrase += substr[i] + " ";

File: tasks/cycles/cycles.js [hunk 3]
```diff
@@ -11,37 +11,85 @@
...
-  };
+    const personIncomes = {
+        salary: 1985,
+        rent: -600,
+        interestOnDeposit: 250,
+        otherExpences: -300
+    };
+    let income = 0;
+    for (let property in personIncomes) {
+        income = income + personIncomes[property];
```
Comment (dshaplyko) [inline tasks/cycles/cycles.js:90]: income += personIncomes[property];

Comment (IrynaKazei) [reply tasks/cycles/cycles.js]: Many thanks. Got it.
Fixed comments.

**Pull request merged**